index.js

```diff
@@ -18,4 +19,6 @@ module.exports = function (file, args, options) {
 			resolve({stdout: stdout, stderr: stderr});
 		});
 	});
+	promise.child = child;
```

Comment:
Ok, if you prefer.  Should I copy all of the methods on the promise instance, or just `then` and `catch` (in case the user has set another implementation as the global `Promise`)?